crates/hdk/src/link.rs

```diff
@@ -33,16 +32,15 @@ pub use hdi::link::*;
 /// See [ `get_details` ] and get for more information about CRUD
 /// See [ `get_links` ] and [ `get_link_details` ] for more information about filtering by tag
 ///
-/// Generally links and CRUDs _do not interact_ beyond the fact that links need entry hashes to
+/// Generally links and CRUDs _do not interact_ beyond the fact that links need hashes to
 /// reference for the base and target to already exist due to a prior create or update.
-/// The entry value only needs to exist on the DHT for the link to validate, it doesn't need to be
+/// The referenced data only needs to exist on the DHT for the link to validate, it doesn't need to be
 /// live and can have any combination of valid/invalid crud actions.
```

Comment:
A link needs to exist on the DHT but doesn't need to be live. What does "live" mean?

Reply:
@jost-s live is a DHT concept, is this the best place to explain it?

Reply:
Is this a rhethorical question? =D

Reply:
@jost-s it doesn't seem like a good place to document it to me, i don't see ppl going to the link docs to understand how the DHT works

Reply:
Sure, agreed